### PR TITLE
Fix example for QueryObject::postFetch()

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: docs/en/optimizing-query-objects.md [hunk 1]
```diff
@@ -56,12 +56,14 @@ class ArticlesQuery extends QueryObject
 
   public function postFetch(Queryable $repository, \Iterator $iterator)
   {
-    $ids = array_keys(iterator_to_array($iterator, TRUE));
```
Comment (enumag) [inline docs/en/optimizing-query-objects.md]: Tiis didn't work for me beacuse the entities were not indexed by their ids. Do I have to enable it somehow or is it not possible?

Comment (fprochazka) [inline docs/en/optimizing-query-objects.md]:
```php
 ->select('article')->from(Article::class, 'article', 'article.id')
```

@enumag it's the third argument of `->from`

Comment (fprochazka) [inline docs/en/optimizing-query-objects.md]: @enumag also, it can be specified even in joins

Comment (enumag) [inline docs/en/optimizing-query-objects.md]: Which is missing in this example. Ok, I'll change it.

Comment (fprochazka) [inline docs/en/optimizing-query-objects.md]: It's here

snimek obrazovky porizeny 2015-10-05 15 18 34

Comment (enumag) [inline docs/en/optimizing-query-objects.md]: Right. I was looking to a bad place.

Comment (enumag) [inline docs/en/optimizing-query-objects.md]: I removed this change.

**Pull request merged**